content/guidelines/administration.md

```diff
@@ -6,14 +6,110 @@ author: "Johan Ljunggren"
 weight: 6
 ---
 
-### Create GitHub repository
+### Security
 
-Not written.
+To be administrator you must have 2FA enabled on accounts that give you
+access to DSC Community resources, for example your GitHub account and
+the account that access DSC Community Azure DevOps organization.
 
-### Create Azure DevOps project
+### GitHub
 
-Not written.
+#### Create GitHub repository
+
+To create a new GitHub repository in the DSC Community organization, browse
+to https://github.com/dsccommunity and click on _New_.
+
+All repositories should always be added with **MIT licensing**.
+
+>**NOTE:** To be able create repositories you must be an owner of the
+>GitHub DSC Community organization. See [GitHub DSC Community organizational owners](https://github.com/orgs/dsccommunity/people?query=role%3Aowner).
+
+##### Create repository Personal Access Token (PAT) to use with the pipeline
+
+The GitHub repository must first exist to be able to do this. Log in to
+the @dscbot account (the DSC Community GitHub account) and from there
+take out a Personal Access Token for the the **specific repository**.
+
+**Do NOT EVER create a PAT for the entire organization!**
```

Comment:
Organisations can't have PAT. Only OAuth token I think.
We are using the DSC Bot account to have a PAT.
Perms are directly coming from the DSC Bot account, so it's push perms on all repos.
We use the PAT to create the PR & to create the GH releases.

Reply:
Discussing in Slack. Wait for the conclusion of that discussion so I know how to change this.

Reply:
Done